database/mongo.py

```diff
@@ -1,28 +1,62 @@
-from os import getenv
 from motor.motor_asyncio import AsyncIOMotorClient as MongoClient
-from motor.motor_asyncio import AsyncIOMotorCollection as MongoCollection
-from database import users
+from bson.json_util import loads,dumps
+import aiofiles, aiofiles.os
+from os import getenv
+from discord.ext import tasks, commands as cmds
+from . import users
+
+
+class MongoCog(cmds.Cog, name='Mongo update script'):
+    def __init__(self, bot):
+        self.bot = bot
+        self.mongo_users = MongoClient(getenv('MONGO_URI')).popskill.user_links
+        self.token = None
+        self.watch.start()
+
+    @tasks.loop(count=1)
+    async def watch(self):
+        async with self.mongo_users.watch([{'$match': {'operationType': {'$in': ['insert', 'update', 'replace']}}}],
+                                          full_document='updateLookup',
+                                          resume_after=self.token) as stream:
+            async for change in stream:
+                entry = change['fullDocument']
+                if 'steam_id' in entry:
+                    await users.add_steam64_id(entry['discord_id'], entry['steam_id'])
+                self.token = stream.resume_token
 
+    @watch.before_loop
+    async def get_token(self):
+        try:
+            async with aiofiles.open('resume', 'rb') as tokenfile:
```

Comment:
Please could you put the `resume` token in the `database` module?

Reply:
Done in 68e8a70